girder/molecules/server/calculation.py

```diff
@@ -79,7 +79,13 @@ def get_calc_vibrational_mode(self, id, mode, params):
 
         projection = {
             'vibrationalModes.modeFrames': {
-                '$slice': [index-1, 1]
+                '$slice': [index, 1]
+            },
+            'vibrationalModes.frequencies': {
+                '$slice': [index, 1]
+            },
+            'vibrationalModes.intensities': {
+                '$slice': [index, 1]
             }
```

Comment:
I did a fresh import, and this was always off by one, if mode/9 was requested index was always 8 for example. When you have time we can take another look, but I think this is the correct fix.